.circleci/config.yml

```diff
@@ -0,0 +1,24 @@
+# This file was generated automatically from conda-smithy. To update this configuration,
```

Comment:
I've been manually deleting this file to reduce clutter since it only has a purpose of the repository is connected to CircleCI.

Reply:
We should `git rm` and  .gitignore this file, then?

Reply:
Yes, although last I checked the .gitignore will have no effect on conda-smithy adding the files as a staged change, unfortunately. I couldn't figure out a way to automatically keep those files out with the current conda-smithy tooling.

Reply:
would we be better of if we had a script that ran conda-smithy, then did the commit rather than using `conda-smithy -c auto`?

Reply:
(note that we also always have to manually sign these commits, as well)